README.md

```diff
@@ -1,18 +1,30 @@
-## 👋 Welcome!
+# Conda GitHub Org Infrastructure
+
+Centralized location for infrastructure codes as well as any infrastructure related inquiries or discussions.
```

Comment:
Maybe change the beginning of this sentence to "This is a centralized location for..."

Reply:
✔️